bobweb/bob/error_handler.py

```diff
@@ -7,22 +7,45 @@
 from telegram.constants import ParseMode
 from telegram.ext import ContextTypes
 
-from bobweb.bob import database
+from bobweb.bob import database, message_board_service
+from bobweb.bob.message_board import MessageBoard
 from bobweb.bob.resources.unicode_emoji import get_random_emoji
 
 logger = logging.getLogger(__name__)
 
 
-async def error_handler(update: object, context: ContextTypes.DEFAULT_TYPE) -> None:
+async def unhandled_bot_exception_handler(update: object, context: ContextTypes.DEFAULT_TYPE) -> None:
     """
     General error handler for all unexpected errors. Logs error, notifies user by replying to the message
     that caught the error and sends traceback to the developer chat.
+
+    TODO: It might be best to first ask the user if contents of the error message can be shared to the developers either
+    TODO: anonymously or with the users details. And only after that would the error be sent to the developers chat with
+    TODO: appropriate level of details.
```

Comment:
This would make sense. Also, not good practice to leave todo's like this. Maybe raise an issue instead.

Reply:
Tästä tehty issue https://github.com/M4R774/bobweb2/issues/277